src/store/mutations.js

```diff
@@ -234,6 +234,9 @@ export default {
     state.isTkeySeedPhraseInputRequired = payload
   },
   setLoginInProgress(state, payload) {
+    if (typeof window != 'undefined') {
+      window.loginInProcess = payload
```

Comment:
the variable name is different.. loginInProgress.!!

Reply:
Ohh shitt!! My bad